ui/app/app.js

```diff
@@ -243,7 +245,9 @@ App.prototype.renderAppBar = function () {
     }, [
 
       h('.app-header.flex-row.flex-space-between', {
-        style: {},
+        className: classnames({
+          'app-header--initialized': !isOnboarding,
```

Comment:
i thought this would override the three classes above `.app-header.flex-row.flex-space-between`, is that right?

Reply:
Doesn't override, but adds to the class list.
Initializing:
![image](https://user-images.githubusercontent.com/8051479/34855470-b8b2cb94-f6f4-11e7-97a2-bbf7d8b5252c.png)

Initialized:
![image](https://user-images.githubusercontent.com/8051479/34855485-d2f05346-f6f4-11e7-9184-19f0c222b6d1.png)